core/pythonAction/pythonaction.py

```diff
@@ -61,7 +61,7 @@ def run():
     result = None
     try:
         exec(flask.g, namespace)
```

Comment:
Actually, I think this statement in particular is not needed (subsumed by the following exec), which is really a bug and not only an optimization issue. Good catch!